src/transformers/models/mbart/modeling_mbart.py

```diff
@@ -582,15 +582,15 @@ def dummy_inputs(self):
             :meth:`transformers.PreTrainedTokenizer.encode` and :meth:`transformers.PreTrainedTokenizer.__call__` for
             details.
 
-            `What are input IDs? <../glossary.html#input-ids>`__
+            `What are decoder input IDs? <../glossary.html#decoder-input-ids>`__
 
             MBart uses a specific language id token as the starting token for :obj:`decoder_input_ids` generation that
             varies according to source and target language, *e.g.* 25004 for `en_XX`, and 25003 for `de_DE`. If
             :obj:`past_key_values` is used, optionally only the last :obj:`decoder_input_ids` have to be input (see
             :obj:`past_key_values`).
 
             For translation and summarization training, :obj:`decoder_input_ids` should be provided. If no
-            :obj:`decoder_input_ids` is provided, the model will create this tensor by shifting the :obj:`input_ids` to
+            :obj:`decoder_input_ids` is provided, the model will create this tensor by shifting the :obj:`labels` to
```

Comment:
this was actually correct - for mBART if no `input_ids` **and** no `labels` are provided -> then `decoder_input_ids` are shifted according to the `input_ids` => this corresponds to mBART's denoising pre-training